Plugins/Wox.Plugin.Everything/Main.cs

```diff
@@ -198,6 +198,43 @@ public List<Result> LoadContextMenus(Result selectedResult)
                 }
             }
 
+            var icoPath = (record.Type == ResultType.File) ? "Images\\file.png" : "Images\\folder.png";
+            contextMenus.Add(new Result
+            {
+                Title = "Copy as path",
```

Comment:
It will be very nice if you take L10n into consideration. You can use other plugin, like [shell plugin](https://github.com/Wox-launcher/Wox/blob/13e6c5eefcc669b9abc93d84b569226afe679e9f/Plugins/Wox.Plugin.Shell/Main.cs#L323) as an example. 
```suggestion
                Title = _context.API.GetTranslation("wox_plugin_everything_copy_as_path"),
```

Please remember to update the corresponding file(s) in the `Languages` folder